manifests/1.3.1/opensearch-dashboards-1.3.1.yml

```diff
@@ -10,3 +10,6 @@ components:
   - name: OpenSearch-Dashboards
     repository: https://github.com/opensearch-project/OpenSearch-Dashboards.git
     ref: '1.3'
+  - name: functionalTestDashboards
+    repository: https://github.com/opensearch-project/opensearch-dashboards-functional-test.git
+    ref: tags/1.3.0
```

Comment:
shall we 1.3 branch or main? I remember once 1.3.1 is officially release, we will create tag 1.3.1 and then update here.

Reply:
Do you plan on doing a version bump ? For this repo it doesn't get bundled so I thought I'd use the `tags/1.3.0`, I can make this `1.3`

Reply:
nvm I think someone backported a improvement for flakiness. Let's use the branch. updated.